### ISSUE TITLE
Installation of flowWorkspace failed in R.

### ISSUE THREAD

**Issue by staedlern** [RGLab/flowWorkspace#265]
Dear Greg

I faced problems installing the package on R.3.5.1 following Bioconductor instructions:

if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("flowWorkspace", version = "3.8")

![image](https://user-images.githubusercontent.com/8275254/48265215-3db89100-e42c-11e8-8d08-1ff7aabd4a1d.png)
![capture](https://user-images.githubusercontent.com/8275254/48265256-588b0580-e42c-11e8-955e-e1aad8bfc1b8.JPG)

**Comment by SamGG** [comment]
Hi,
Just for information, no problem on Win7x64, R351. In your log, I don't understand why bioconductor is still mentioned. As I don't know your R installation level, may I suggest you the following procedure? Close all your R or RStudio sessions, start RStudio, close the current project, open a new empty project, then do the installation. This will ensure that no package is loaded before installation. I did the update from 3.7 to 3.8 on monday without any trouble. I used simply `BiocManager::install()`, which seems to update all the installed packages.
Best.
![2018-11-09_144900](https://user-images.githubusercontent.com/3333348/48266507-dac8f900-e42f-11e8-99c0-6972497f7481.png)
